apps/web/pages/organizations/[id]/checkout.page.tsx

```diff
@@ -0,0 +1,104 @@
+import {
+  PayPalButtons,
+  PayPalButtonsComponentProps,
+  PayPalScriptProvider,
+  ReactPayPalScriptOptions,
+} from '@paypal/react-paypal-js'
+import { useRouter } from 'next/router'
+import { useMutation } from 'urql'
+
+import { PageHeading } from '../../../frontend/components/pageHeading'
+import { ProtectedPage } from '../../../frontend/components/protectedPage'
+import { graphql } from '../../../frontend/generated/gql'
+
+const organizationPaypalSubscriptionIdCreateMutationDocument = graphql(`
+  mutation organizationPaypalSubscriptionIdCreate($organizationId: ID!, $returnUrl: String!, $cancelUrl: String!) {
+    organizationPaypalSubscriptionIdCreate(
+      organizationId: $organizationId
+      returnUrl: $returnUrl
+      cancelUrl: $cancelUrl
+    )
+  }
+`)
+
+// const PayPalButtonWrapper = () => {
+//   const [{ options }, dispatch] = usePayPalScriptReducer()
+//   const [currency, setCurrency] = useState(options.currency)
+
+//   function onCurrencyChange({ target: { value } }: any) {
+//     setCurrency(value)
+//     dispatch({ type: 'resetOptions', value: { ...options, currency: value } })
+//   }
+
+//   return (
+//     <>
+//       <select value={currency} onChange={onCurrencyChange}>
+//         <option value="USD">US Dollar</option>
+//         <option value="EUR">Euro</option>
+//       </select>
+
+//       <PayPalButtons createOrder={createOrder} onApprove={onApprove} style={{ layout: 'vertical' }} />
+//     </>
+//   )
+// }
+
+const PayPalPage = (): JSX.Element => {
```

Comment:
Let's use a name that matches the route

```suggestion
const CheckoutPage = (): JSX.Element => {
```